esp/bindings/SOAP/Platform/soapbind.hpp

```diff
@@ -148,6 +151,12 @@ class esp_http_decl CSoapRequestBinding : public CSoapComplexType,
     void setThunkHandle(void * val){thunk_=val;}
     void * getThunkHandle(){return thunk_;}
 
+    void setConnectTimeOutMs(unsigned timeout){connectTimeoutMs = timeout;}
```

Comment:
you can declare your function definitions as "override" when the function overrides a base virtual function (even if your implementation is in-line). If these functions aren't overriding a base implementation, then ignore, but I thought they did.
http://en.cppreference.com/w/cpp/language/override